Sources/App/Views/Author/AuthorShow+View.swift

```diff
@@ -67,8 +76,15 @@ enum AuthorShow {
                             .li(
                                 .a(
                                     .href(package.url),
-                                    .h4(.text(package.title)),
-                                    .p(.text(package.description))
+                                    .class("two_column"),
+                                    .div(
+                                        .h4(.text(package.title)),
+                                        .p(.text(package.description))
+                                    ),
+                                    .p(
+                                        .span(.id("star_text")),
```

Comment:
This is looking great, but I'd just make one small change. Elements with an `id` should be unique, like the `package_list` one on the `ul` above. There's only ever one of those elements in the document, so it's a candidate for an `id`. This span is repeated multiple times, so should be a `class`.

```suggestion
                                        .class("stars"),
```

Also, I had a quick play and even though I suggested it, we can get away without that extra `span`. See the CSS suggestion I'm making along with this one for a solution.